tarteaucitron.services.js

```diff
@@ -2273,3 +2273,47 @@ tarteaucitron.services.matomo = {
         tarteaucitron.addScript(tarteaucitron.user.matomoHost + 'piwik.js', '', '', true, 'defer', true);
     }
 };
+
+// Hotjar
+
+/*
+    1. Set the following variable before the initialization :
+
+    tarteaucitron.user.hotjarId = YOUR_WEBSITE_ID;
+    tarteaucitron.user.HotjarSv = XXXX; // Can be found in your website tracking code as "hjvs=XXXX"
+
+    2. Push the service :
+
+    (tarteaucitron.job = tarteaucitron.job || []).push('hotjar');
+
+    3. HTML
+    You don't need to add any html code, if the service is autorized, the javascript is added. otherwise no.
+
+
+ */
+tarteaucitron.services.hotjar = {
+    "key": "hotjar",
+    "type": "analytic",
+    "name": "Hotjar",
+    "uri": "https://help.hotjar.com/hc/en-us/sections/115003204947-What-is-Hotjar-",
+    "needConsent": true,
+    "cookies": [],
```

Comment:
Add list of cookies
https://help.hotjar.com/hc/en-us/articles/115011789248-Hotjar-Cookies